### PR TITLE
Correct license type

### PR DESCRIPTION
**Pull request by bclinkinbeard**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "distance transform"
   ],
   "author": "Vladimir Agafonkin",
-  "license": "ISC",
+  "license": "BSD",
```
Comment (mourner) [inline package.json]: I believe the correct one for the text is `BSD-2-Clause`: https://spdx.org/licenses/

Comment (bclinkinbeard) [reply package.json]: Fixed

**Comment by bclinkinbeard** [comment]
Thanks! Could you publish a new version with the change?

**Comment by mourner** [comment]
Let me do some more changes and publish a new semver-major tomorrow.

**Comment by bclinkinbeard** [comment]
A new major will unfortunately not help me as tiny-sdf is a transitive dependency. Could you publish a patch with just the LICENSE added?

**Comment by mourner** [comment]
@bclinkinbeard it's tricky because I merged more substantial changes after your PR. Would it help if I updated the transitive dep too (I'm assuming GL JS?)

**Comment by bclinkinbeard** [comment]
No, @deck.gl/layers. You should be able to checkout the merge commit and publish that without anything that came after it. https://stackoverflow.com/questions/24691314/npm-publish-patch-for-earlier-major-version

**Comment by mourner** [comment]
@bclinkinbeard OK, published it as 1.1.1!

**Comment by bclinkinbeard** [comment]
Awesome, thank you!!

**Pull request merged**